src/setup.py

```diff
@@ -203,9 +203,12 @@ class MetadataReader(object):
     _CONNECT_TIMEOUT_IN_SECONDS = 0.3
     _RESPONSE_TIMEOUT_IN_SECONDS = 0.5
     _REQUEST_TIMEOUT = (_CONNECT_TIMEOUT_IN_SECONDS, _RESPONSE_TIMEOUT_IN_SECONDS)
+    _TOKEN_REQUEST = "latest/api/token"
+    _TOKEN_TTL_SECONDS = 21600 # 6 hours
```

Comment:
the code looks like the same as above, what's the difference? can we abstract them?

Reply:
it is in setup.py, the other is in plugin itself.  the codes are duplicated, but it is hard to pull out a common module. 
I just follow previous way.